examples/step-18/doc/results.dox

```diff
@@ -12,8 +12,8 @@ much more reasonable time.
 
 If run, the program prints the following output, explaining what it is
 doing during all that time:
-@code
-$ time make run
+@verbatim
+\$ time make run
 [ 66%] Built target step-18
 [100%] Run step-18 with Release configuration
```

Comment:
I suspect that you still need the backslash in front of `step-18` here. The substitution of `step-XX` to `@ref step_XX "step-XX"` is made by the doc/doxygen/scripts/filter script, which does not know whether you're in verbatim mode or not. The substituted text would usually be made into a reference by doxygen, but it will leave the text in place inside `@verbatim` I suppose and it's going to look funny.